dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -205,7 +205,7 @@ RUN cd /usr/share/heartbeat/.node \
     && curl ${NODE_DOWNLOAD_URL} | tar -xJ --strip 1 -C node \
     && chmod ug+rwX -R $NODE_PATH
 
-# Install synthetics as a regular user, installing npm deps as root odesn't work
+# Install synthetics as a regular user, installing npm deps as root doesn't work
 RUN chown -R {{ .user }} $NODE_PATH
 USER {{ .user }}
```

Comment:
So, it's not required in the rest of the places, only at the end of the Dockerfile?

Reply:
iiuc it's the final user that the container runs as which is checked by this setting. Some other components need to be installed as the root user prior to pivoting to an unprivileged user.